Misc/NEWS

```diff
@@ -327,6 +327,9 @@ Extension Modules
 Library
 -------
 
+- bpo-30378: Fix the problem that logging.handlers.SysLogHandler cannot
```

Comment:
Not sure having this in the patch is a good idea - may cause unnecessary merge conflicts.